src/Compiler/Service/BackgroundCompiler.fsi

```diff
@@ -102,6 +102,20 @@ type internal IBackgroundCompiler =
         userOpName: string ->
             Async<FSharpProjectOptions * FSharpDiagnostic list>
 
+    abstract GetProjectSnapshotFromScript:
```

Comment:
Oh, we should probably mark this as experimental.

Reply:
Yup, you mean at the `service.fsi` layers right?

Reply:
Yep, I put the comment in the wrong file.

Reply:
Alright, that part is done!